morebits.css

```diff
@@ -8,22 +8,60 @@
  * The latest development source is available at [https://github.com/wikimedia-gadgets/twinkle/blob/master/morebits.css].
  */
 
+/* Define CSS variables */
+html {
+	--morebits-color-status-status: #4682B4;
+	--morebits-color-legend: #31628F;
+	--morebits-color-tooltip: #0000cd;
+	--morebits-color-info: #228B22;
+	--morebits-color-warning: #FF4500;
+	--morebits-color-titlebar-links: #3062AD;
+	--morebits-bgcolor-dialog: #F0F8FF;
+	--morebits-bgcolor-titlebar: #BCCADF;
+}
+
+@media screen {
+	html.skin-theme-clientpref-night {
+		--morebits-color-status-status: #A6C4DE;
+		--morebits-color-legend: #6B9DCC;
+		--morebits-color-tooltip: #7A7AFF;
+		--morebits-color-info: #6BDB6B;
+		--morebits-color-warning: #FFC7B3;
```

Comment:
Looks too whitish red. Not attention-grabbing as this color is supposed to be. 

(To test this on testwiki, in XFD module try to list a page at RM/TR - it fails and an error with this color is shown.)

Reply:
#FF6D38 seemed to work better.